src/services/blocks/BlocksService.ts

```diff
@@ -70,7 +74,8 @@ export class BlocksService extends AbstractService {
 			checkFinalized,
 			queryFinalizedHead,
 			omitFinalizedTag,
-		}: FetchBlockOptions
+		}: FetchBlockOptions,
+		that: That
```

Comment:
I honestly think this is a code smell. The `metaConstsCache` should belong to the `BlocksService`. I know there was some issues with this but we should be able to figure something once we get everything else straigtened out.

This current design means for every controller that uses the `BlocksService`, it will need to have its own, redundant `metaconstsCache`, which seems like a waste of memory since sidecar is designed to only connect to one chain.

Reply:
Fixed!